### DISCUSSION TITLE
Transparency with mesh instancing?

### DISCUSSION THREAD

**Discussion by Semihazah** [bevyengine/bevy#8766] [Q&A]
I'm using the instancing example here https://github.com/bevyengine/bevy/blob/main/examples/shader/shader_instancing.rs.
Setting the alpha value in the shader does nothing, so I suspect I need to change something about the pipeline, but I don't know what.

**Comment by insberr** [comment]
I am also having this issue. I also want shadows to work. If anyone knows how to make the alpha and shadows work that would be great!

**Comment by gorynich2048** [comment]
You need to enable alpha blending.
Add this line in the `specialize` function:
```rs
descriptor.fragment.as_mut().unwrap().targets[0].as_mut().unwrap().blend = Some(BlendState::ALPHA_BLENDING);
```
Or add `MeshPipelineKey::BLEND_ALPHA` to the pipeline's key.
